soh/soh/Enhancements/randomizer/3drando/hints.cpp

```diff
@@ -533,11 +533,11 @@ int32_t getRandomWeight(int32_t totalWeight){
 static void DistributeHints(std::vector<uint8_t>& selected, size_t stoneCount, std::vector<HintDistributionSetting> distTable, uint8_t junkWieght, bool addFixed = true){
   int32_t totalWeight = junkWieght; //Start with our Junk Weight, the natural chance of a junk hint
 
-  for (HintDistributionSetting setting: distTable){ //Gather the wieghts of each distribution and, if it's the first pass, apply fixed hints
-    totalWeight += setting.weight;                  //Note that PlaceHints will set weights of distributions to zero if it can't place anything from them
+  for (uint8_t c=0; c < distTable.size(); c++){ //Gather the wieghts of each distribution and, if it's the first pass, apply fixed hints
```

Comment:
```suggestion
  for (size_t c = 0; c < distTable.size(); c++){ //Gather the wieghts of each distribution and, if it's the first pass, apply fixed hints
```

Reply:
Just don't want anny issues with c overflowing

Reply:
Changed a couple of others I quickly found while i was thinking about it